ADImport/Properties/AssemblyInfo.cs

```diff
@@ -14,6 +14,6 @@
 [assembly: AllowPartiallyTrustedCallers]

```

Comment:
Please add "Xperience" to the AssemblyTitle and AssemblyDescription attributes above.